Mage.Client/src/main/java/mage/client/draft/DraftPanel.java

```diff
@@ -821,8 +821,8 @@ private void initComponents() {
 
      private void btnQuitTournamentActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_btnQuitTournamentActionPerformed
```

Comment:
Does the comment on here mean that this function is auto-generated?

Reply:
The .form file contains nothing related to this function, except that it's called when pressing the quit-button, which I think means it's not auto-generated.

I noticed that there's a similar dialog in tournamentpanel, and I did the same yes/no switch to it for consistency's sake.

Reply:
It's GUI's event: NetBeans generates empty lines for it like start and end (see `//GEN-FIRST` and `//GEN-LAST`). All other code between that lines are user defined and keeps on GUI editing. So it's ok to change code in the events.

P.S. IntelliJ doesn't support NetBeans's auto-generates blocks, so it allows to edit it. If you open file with NetBeans then auto-generated blocks will be read only.

Reply:
Thanks for the info! And for merging this. It's always been a bummer when someone has hit the double spacebar.